### PR TITLE
New theme colors and button style

### LINKED ISSUE #496: Set right colors in the `Button` style component of the Website

### PR DESCRIPTION
**Pull request by cedoor**
## Related Issue

Closes #496

## Does this introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: apps/website/src/styles/components/Button.ts [hunk 1]
```diff
@@ -14,34 +13,53 @@ const Button = {
...
         solid: (props: StyleFunctionProps): SystemStyleObject => {
             const { colorScheme: c } = props
 
+            if (c === "primary") {
+                const bgGradient = "linear(to-r, primary.500, primary.800)"
+                const color = "white"
+
+                return {
+                    bgGradient,
+                    color,
```
Comment (vplasencia) [inline apps/website/src/styles/components/Button.ts:28]:
```suggestion
                    color,
                    transitionDuration:"200ms",
                    transitionTimingFunction: "linear",
```

What about adding something like this here?

**Review by vplasencia — APPROVED** [review]

File: apps/website/src/styles/components/Button.ts (hunk 1 above)
Comment (vplasencia) [reply apps/website/src/styles/components/Button.ts:28]: Maybe do the same with all the other buttons?

Comment (cedoor) [reply apps/website/src/styles/components/Button.ts:28]: I like it, I've added them

**Review by vplasencia — APPROVED** [review]
Great work!

**Pull request merged**